pages/ycrv/holdings.tsx

```diff
@@ -97,11 +97,17 @@ function	Holdings(): ReactElement {
 	}, []);
 
 	const	latestCurveFeesValue = useMemo((): number => {
-		if (curveWeeklyFees?.weeklyFeesTable?.[0]?.rawFees > 0) {
```

Comment:
Found a potential issue here where `weeklyFeesTable` can be `undefined`

<img width="547" alt="Screenshot 2023-05-22 at 14 25 24" src="https://github.com/yearn/yearn.fi/assets/78794805/9b345197-ffda-4af7-9586-6d2f712a2a27">